### ISSUE TITLE
Is this project abandoned?

### ISSUE THREAD

**Issue by kulbhushanchand** [Pseudonium/Obsidian_to_Anki#613]
The last commit and release was 8 months ago (Feb 1, 2024).
There are many open pull requests but no action on them. 
Can the owner please confirm if there are plans to maintain the project in future?
This is an awesome and to the best of my knowledge best obsidian-anki plugin that offers support for any note type.

**Comment by envico801** [comment]
I agree, it seems a bit abandoned, personally I have been making my own changes to my version of the plugin to adapt it to what I need.

**Comment by mattcoleanderson** [comment]
@ShootingKing-AM @Pseudonium What is blocking further development? Is it time? If so, now that this project is in it's own Organization can we start adding more maintainers to the project? 

Right now this highly used project is falling prey to [Bus Factor](https://en.wikipedia.org/wiki/Bus_factor), that is should `x` number of team member be run over by a bus, would it put the project in jeopardy? Right now that bus factor is **2**.

The best way to keep your hard work going is to trust the community to carry on your vision, its a hard thing to do, but they can do it. I say pick 3-5 people as maintainers, particularly people who have contributed in the past, and pass the torch. Let them make changes without oversight. This is the only way FOSS projects stay alive.

**Comment by Eirikfesker** [comment]
we need updates, my whole study program runs on this awesome project which allowed me to make flashcards easier then ever. but now it doesnt show up in the ribbon bar anymore and i cant export new cards and make folde rnotes and so on -  if its time and money make a donation button!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

**Comment by ShootingKing-AM** [comment]
Sadly i convinced @Pseudonium to make an org and give essential permissions to me. But hes kinda reluctant to do that. Adding new maintainers on this repo is also not possible at this point. Maybe hes busy irl. At this point, i think the way forward is to continue development work on my fork https://github.com/ShootingKing-AM/Obsidian_to_Anki and leave this repo aside.

**Comment by ShootingKing-AM** [comment]
related #447

**Comment by envico801** [comment]
Many users are mistakenly ending up in this repository (Pseudonium/Obsidian_to_Anki) while looking for help. It would be helpful if @Pseudonium could:

1. Update references to point to the new repository that is actively maintaining updates.
2. Add a note in this repository's `README.md` to clarify its status as an archive or reference point. This would guide users to the newer repository for the latest solutions while allowing them to explore this repository for older solutions.

This change would reduce confusion and improve the user experience for those seeking support.

![listed plugin on obsidian](https://github.com/user-attachments/assets/31209a2c-7796-4897-984b-361cf8c37198)

**Comment by mattcoleanderson** [comment]
> Many users are mistakenly ending up in this repository (Pseudonium/Obsidian_to_Anki) while looking for help.

@envico801 thats the thing though. They aren't mistakenly ending up here.

@ShootingKing-AM's fork doesn't seem to be endorsed by @Pseudonium. This is still the official plugin.

I do think that @ShootingKing-AM  should attempt to get his fork published to the store though. Idk if they would approve it though
